### PR TITLE
Update build and test deps to the latest version

### PR DESCRIPTION
**Pull request by JanuszL**
- updates libjpeg-turbo
- updates libtorpobuff
- updates OpenCV
- updates FFmpeg
- updates libvorbis
- updates liblmdb

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It updates build and test deps to the latest version

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     updates build and test deps to the latest version
 - Affected modules and functionalities:
     aarch64 build
     x86_64 build
     QNX build
     conda build
     tests
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     NA

**JIRA TASK**: *[NA]*

### CODE REVIEW

File: conda/recipe/meta.yaml [hunk 1]
```diff
@@ -62,8 +62,8 @@ requirements:
   host:
     - python
     - future >=0.17.1
-    - protobuf =3.12.4
-    - libprotobuf-static =3.12.4
+    - protobuf =3.13.0
+    - libprotobuf-static =.13.0
```
Comment (klecki) [inline conda/recipe/meta.yaml:66]:
```suggestion
    - libprotobuf-static = 3.13.0
```
:duck:

Comment (JanuszL) [reply conda/recipe/meta.yaml]: Done

**Review by awolant — APPROVED** [review]

File: qa/setup_packages.py [hunk 2]
```diff
@@ -313,12 +313,12 @@ def get_pyvers_name(self, url, cuda_version):
...
 
-all_packages = [PlainPackage("opencv-python", ["4.2.0.32"]),
+all_packages = [PlainPackage("opencv-python", ["4.4.0.42"]),
                 CudaPackage("cupy",
-                        { "100" : ["7.5.0"] },
+                        { "100" : ["7.8.0"] },
                         "cupy-cuda{cuda_v}"),
                 CudaPackage("mxnet",
-                        { "100" : ["1.5.1"] },
+                        { "100" : ["1.7.0"] },
```
Comment (klecki) [inline qa/setup_packages.py:321]: On the mxnet page I see 1.6 as the latest.

Comment (JanuszL) [reply qa/setup_packages.py:321]: https://pypi.org/project/mxnet-cu100/ - it shows 1.7

**Review by klecki — APPROVED** [review]

**Pull request merged**